support-frontend/assets/components/forms/customFields/validated.jsx

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React, { type Node } from 'react';
+
+import { type Option } from 'helpers/types/option';
+
+// ----- Types ----- //
+export type PropsForHoc = {
+  error: Option<string>,
+  valid: Option<string>,
+};
+
+type Props = PropsForHoc & {
+  htmlFor: Option<string>,
+  children?: Option<Node>,
+};
+
+// ----- Component ----- //
+
+const getClassName = (error: Option<string>, valid: Option<string>) => {
+  if (error) {
+    return 'component-form-error';
+  } else if (valid) {
+    return 'component-form-valid';
+  }
+  return null;
+};
+
+function Validated({
+  error,
+  valid,
+  htmlFor,
+  children,
+}: Props) {
+  const Element = htmlFor ? 'label' : 'div';
+
+  return (
+    <div className={getClassName(error, valid)}>
+      {children && children}
+      <Element
+        aria-hidden={!error}
+        aria-atomic="true"
+        aria-live="polite"
+        htmlFor={htmlFor}
+        className="component-form-error__error"
+      >
+        {(error === 'Temporary COVID message') && (
+          <li className="component-form-error__summary-error">
+            The address and postcode you entered is outside of our delivery area. You may want to
```

Comment:
not sure if this is relevant to digi?

Reply:
It's not but this component is a generic one that can be used elsewhere

Reply:
I don't fully understand, if it's got paper HD stuff in it it's not really generic?  If it was, any product specific stuff should be passed in as a parameter.  Happy to defer though as I'm not totally up on structuring client side stuff.

Reply:
Yes it's come directly from the error component:
https://github.com/guardian/support-frontend/blob/6de367509e0813e5aaae7852693e10d1f828b5e3/support-frontend/assets/components/forms/customFields/error.jsx#L35
It is a hack that was put in place to handle the Covid message without having to do a major refactor, with any luck we can remove it from both places in the not too distant future.